extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/CodeAuthenticationMechanism.java

```diff
@@ -364,7 +364,8 @@ public Throwable apply(Throwable tInner) {
     }
 
     private String generateInternalIdToken(OidcTenantConfig oidcConfig) {
-        return Jwt.claims().sign(KeyUtils.createSecretKeyFromSecret(oidcConfig.credentials.secret.get()));
+        String tenant = oidcConfig.tenantId.orElse("default");
+        return Jwt.claims().issuer(tenant).sign(KeyUtils.createSecretKeyFromSecret(oidcConfig.credentials.secret.get()));
```

Comment:
As I said it is unnecessary

Reply:
As long as I can get the `OidcSession` before my `SecurityIdentityAugmentor` is called: otherwise it'll be a bogus `SecurityIdentity`

Reply:
@FroMage in `SecurityIdentityAugmentor` the tenant id is set as a `SecurityIdentity.attribute` - the injection does not work at its level; in custom `TenantResolver` it will be an routing context attribute; request scoped injection won't work there either

Reply:
I just tried it, and I'm not getting any attribute set to the tenant ID, is it normal?